.github/release.yml

```diff
@@ -0,0 +1,26 @@
+changelog:
+  exclude:
+    authors:
+      - dependabot
+  categories:
+    - title: Breaking Changes
+      labels:
+        - "breaking-change"
+    - title: Bug Fixes
+      labels:
+        - "bug"
+    - title: New Features
+      labels:
+        - "enhancement"
+    - title: Configuration
+      labels:
+        - "configuration"
+    - title: Documentation
+      labels:
+        - "documentation"
+    - title: Dependency Updates
+      labels:
+        - "dependencies"
+    - title: Other Changes
```

Comment:
I sometimes call this "Internal Changes", like, not affecting consumers/operators